projects/vdk-core/src/vdk/api/plugin/plugin_input.py

```diff
@@ -214,6 +213,16 @@ def ingest_payload(
         """
         pass
 
+    def process_payload(self, payload: List[dict]) -> List[dict]:
+        """
+        Do some processing on the ingestion payload before passing it to the actual
```

Comment:
What kind of processing .Can you add example usage?

Reply:
Example added, thanks